### PR TITLE
Run all tests and verifications via fastlane, on releases and Circle

### PR DESCRIPTION
**Pull request by KrauseFx**
Please don't squash, these are 2 changes: 

- Run all tests and verifications via fastlane, on releases and Circle
- Add Brewfile

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -12,6 +12,4 @@ test:
   pre:
     - ulimit -n 400
   override:
-    - bundle exec rspec -r rspec_junit_formatter --format progress --format RspecJunitFormatter -o $CIRCLE_TEST_REPORTS/rspec/junit.xml
```
Comment (KrauseFx) [inline circle.yml]: Oh I can see how the test results are not being stored in the proper Circle location now, to see them at the top of the output

**Review by hjanuschka — APPROVED** [review]

File: circle.yml [hunk 2]
```diff
@@ -12,6 +12,4 @@ test:
...
-    - bundle exec rubocop
-    - find -E . -regex ".*\.(sh|bash)" -not -name "Pods-*" -exec shellcheck {} +
+    - bundle exec fastlane test
```
Comment (lacostej) [inline circle.yml]: should this be the new documented way of testing a PR locally?

Comment (KrauseFx) [inline circle.yml]: Yes it should. Is it something else currently?

Comment (lacostej) [inline circle.yml]: See proposed #8090

**Pull request merged**